test/functional/custom-transform.spec.ts

```diff
@@ -66,62 +125,51 @@ describe("custom transformation decorator", () => {
 
     });
 
-    it("versions and groups should work with @Transform decorator too", () => {
+    it("Non class objects should not break if transform is used", () => {
```

Comment:
Please look at this pull request. It breaks your tests when added. https://github.com/pleerock/routing-controllers/pull/86